temp.py

```diff
@@ -0,0 +1,2 @@
+import pdb
+print("hello")
```

Comment:
**CODE_QUALITY**: Consider removing print statements from the production code. If logging is required, use the logging module instead.
```python
import logging

logging.basicConfig(level=logging.INFO)
logging.info('hello')
```